dor-workflow-service.gemspec

```diff
@@ -17,13 +17,13 @@ Gem::Specification.new do |gem|
   gem.test_files    = gem.files.grep(%r{^(spec)/})
   gem.require_paths = ["lib"]
 
-  gem.add_dependency "activesupport", '>= 3.2.1', '< 4.2.2'
+  gem.add_dependency "activesupport", '>= 3.2.1', '< 5'
   gem.add_dependency "nokogiri", '~> 1.6.0'
   gem.add_dependency "rest-client", '~> 1.7'
   gem.add_dependency "confstruct", '~> 0.2.7'
 
   gem.add_development_dependency "rake"
-  gem.add_development_dependency "rspec"
+  gem.add_development_dependency "rspec", "~> 3.3"
```

Comment:
hmm... i'd be somewhat in favor of comments on any pinning that's not a straight-forward lower bound.  i agree that you definitely want explanation on upper bounds and exact versions, moreso than for "~>" versioning.  and with this specific rspec thing, you can probably assume something like "tests simply don't work with rspec 1.x or 2.x"... but there are lots of cases where i'd like to know why something is pinned to a major release.  e.g., why does argo do this:
gem 'nokogiri', '~> 1.6'

is it because someone started using that version of nokogiri, wanted to make sure nothing broke while they were developing, and forgot to test with a more optimistic specification?  or did they actually find something wrong with other versions?  i might be able to find out from the commit history, but i also might not, and the original developer might not be here anymore, and i might not be confident that test coverage will suss out non-obvious issues with other versions.  not to pick on argo too much or anything.

also, since i'm picking on things that irk me:  my understanding is that these two Gemfile statements would be equivalent:
gem 'nokogiri', '~> 1.5'
gem 'nokogiri', '~> 1.6'

is that true?  if so, that seems a little silly, and completely counterintuitive to me (because then something like "~> 1" would be saying nothing over a plain inclusion, right?).  that's not a productive comment for this thread, just a general complaint about bundler (or a request to disabuse me of my misunderstanding).